frontend/public/components/factory/table.tsx

```diff
@@ -344,6 +347,9 @@ export type VirtualBodyProps<D = any, C = any> = {
   scrollTop: number;
   width: number;
   expand: boolean;
+  getRowId?: (obj) => string;
+  getRowTitle?: (obj) => string;
+  getRowClassName?: (obj) => string;
```

Comment:
We might consider having a single `getRowProps` function that could be used to set any prop on the row.

I'm having trouble finding the PF doc for `TableRow`. I'm curious how `id` and `title` are used on the table. Is `title` a `title` HTML attribute on the `tr` element?

Reply:
yes, thats good ! Updated.

`TableRow` is actually a component from console https://github.com/openshift/console/blob/master/frontend/public/components/factory/table.tsx#L139 which wraps `tr`. `title` is HTML attr, `id` is used as `data-id`